src/Aws/Lambda/Runtime.hs

```diff
@@ -66,17 +71,17 @@ invokeAndRun callback manager lambdaApi event context = do
 invokeWithCallback
   :: Throws Error.Invocation
   => Throws Error.EnvironmentVariableNotSet
-  => Runtime.RunCallback
+  => Runtime.RunCallback c
```

Comment:
```suggestion
  => Runtime.RunCallback context
```

Reply:
Ah, I went through these ones, but somehow they've sneaked in.